randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -163,11 +170,23 @@ Extra - asset_id: 4
   * Layers: default
   * Normal Door to Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
-      Grapple Beam or Movement (Beginner)
+      Any of the following:
+          Grapple Beam or Space Jump
+          # It is possible to jump across by not using full jumps to avoid bonking on the ceiling.
+          Movement (Intermediate)
```

Comment:
what about HIBJs? those can be done from the platform and dont require gravity

Reply:
Some of these gaps are pretty wide. I've never attempted to HBJ over them, not sure how difficult it would be. HBJ are harder tricks in SM imo. I'll add it as a TODO.
![image](https://github.com/randovania/randovania/assets/46821362/42034533-53dd-4532-9081-f8a3fa922186)